frontend/lib/src/components/elements/IFrame/IFrame.test.tsx

```diff
@@ -128,16 +130,17 @@ describe("st.iframe", () => {
       const props = getProps({
         scrolling: true,
       })
-      const wrapper = shallow(<IFrame {...props} />)
-      expect(wrapper.find("iframe").prop("style")).toEqual({})
+      render(<IFrame {...props} />)
+      expect(screen.getByTestId("stIFrame")).toHaveAttribute(
+        "scrolling",
+        "auto"
+      )
     })
 
     it("should set `overflow` to hidden", () => {
       const props = getProps({})
-      const wrapper = shallow(<IFrame {...props} />)
-      expect(wrapper.find("iframe").prop("style")).toEqual({
-        overflow: "hidden",
-      })
+      render(<IFrame {...props} />)
+      expect(screen.getByTestId("stIFrame")).toHaveStyle("overflow: hidden")
```

Comment:
Nit: Could you also check that attribute scrolling doesn't exist or is set to no, whichever is the case